app-misc/vcontrold/vcontrold-0.98.11.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="Daemon for communication with Viessmann Vito heatings"
+HOMEPAGE="https://github.com/openv/vcontrold/"
+SRC_URI="https://github.com/openv/vcontrold/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+man +vclient vsim"
+
+DEPEND="dev-libs/libxml2:2"
+RDEPEND="${DEPEND}"
+BDEPEND="man? ( $(python_gen_any_dep 'dev-python/docutils[${PYTHON_USEDEP}]') )"
```

Comment:
Yes this will make **sure** you get a working docutils package. While in reality no one could install a broken one anymore since the package itself checks if there's a PYTHON_COMPAT set or not.

Reply:
Actually giving it some more thought and considering my own words, there's no need to introduce PYTHON_COMPAT for this ebuild to be sure we have a working docutils installed. I'll update the ebuild.